src/main/java/service/TextingService.java

```diff
@@ -51,6 +64,11 @@ public void textMap(String phone, String mapUrl){
 
         System.out.println(message.getSid());
     }
-}
 
-*/
\ No newline at end of file
+
+    public static void main(String[] args) {
```

Comment:
I assume this is for manual testing?

Reply:
Yeah, I was using it since the button isn't currently showing up in the UI